### PR TITLE
fix defaults set issue and add print

### PR DESCRIPTION
**Pull request by dem108**
# PR into Azure/azureml-examples

## Checklist

I have:

- [x] read and followed the contributing guidelines

## Changes

- fix an issue and use the right way to set default batch endpoint
- add verification logic for default deployment
- print explanation for each step for visibility

fixes #

### CODE REVIEW

**Comment by dem108** [comment]
#assign-reviewer: @SagarikaKengunte @n-iyer

**Comment by dem108** [comment]
#assign-reviewer: @SagarikaKengunte

File: cli/batch-score.sh [hunk 1]
```diff
@@ -8,34 +8,42 @@ export ENDPOINT_NAME="<YOUR_ENDPOINT_NAME>"
...
 # </check_batch_endpooint_detail>
 
+echo "Showing details of the batch deployment"
 # <check_batch_deployment_detail>
 az ml batch-deployment show --name nonmlflowdp --endpoint-name $ENDPOINT_NAME
 # </check_batch_deployment_detail>
 
+echo "Invoking batch endpoint with public URI (MNIST)"
 # <start_batch_scoring_job>
 JOB_NAME=$(az ml batch-endpoint invoke --name $ENDPOINT_NAME --input-path folder:https://pipelinedata.blob.core.windows.net/sampledata/mnist --query name -o tsv)
```
Comment (sunjolia) [inline cli/batch-score.sh:38]: What I meant by this comment on the docs PR: https://github.com/MicrosoftDocs/azure-docs-pr/pull/198470#discussion_r879959605

Is that should this command on line 38 still be using --input-path or should it be using --input and --input-type?

**Comment by dem108** [comment]
Found that PR from forked repo did not have right permission. Closing this one.

**Pull request closed without merging**